scripts/circleci/write_junit_report.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+node ./scripts/tasks/junit "$1" "$2" "$3"
```

Comment:
Could you leave a comment in this script about what these arguments are?

Reply:
sure